.env.template

```
@@ -7,3 +7,9 @@ FLASK_DEBUG=true
 
 # Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
 SECRET_KEY=secret-key
+
+# Add your Trello API Credentials here:
+TRELLO_API_KEY = <FILL ME IN!>
+TRELLO_API_TOKEN = <FILL MEIN!!>
+TODO_LIST_ID = <FILL ME IN!!>
+TRELLO_DONE_LIST_ID = <FILL ME IN!!>
```

Thanks for adding these - it makes setting up your app really simple - but there's one slipped the net!

```suggestion
TRELLO_DONE_LIST_ID = <FILL ME IN!!>
TRELLO_BOARD_ID = <FILL ME IN!!>
```